src/components/Integrations/Shopify/Shopify.js

```diff
@@ -1,16 +1,33 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Helmet } from 'react-helmet';
+import { HardcodedShopifyClient } from '../../../services/shopify-client.doubles';
 
-const Shopify = () => (
-  <>
-    <Helmet>
-      <title>Doppler | Shopify</title>
-    </Helmet>
-    <section style={{ width: '100%', padding: '60px 30px' }}>
-      Panel de control | Integraciones y preferencias avanzadas
-      <h1>Panel de Control - Shopify</h1>
-    </section>
-  </>
-);
+const shopifyClient = new HardcodedShopifyClient();
+
+const Shopify = () => {
+  const [shopName, setShopName] = useState('');
+  useEffect(() => {
+    const getData = async () => {
+      const result = await shopifyClient.getShopifyData();
+      if (result.length) {
+        setShopName(result[0].shopName);
+      }
+    };
+    getData();
+  }, []);
+
+  return (
+    <>
+      <Helmet>
+        <title>Doppler | Shopify</title>
+      </Helmet>
+      <section style={{ width: '100%', padding: '60px 30px' }}>
+        Panel de control | Integraciones y preferencias avanzadas
+        <h1>Panel de Control - Shopify</h1>
+        {shopName.length ? 'Shopify conectado a tienda: ' + shopName : 'Shopify desconectado'}
```

Comment:
I think that you are missing some possible states:
- [ ] Undetermined (when ajax call did not finished yet)
- [x] When there is not a shop
- [x] When there is a shop
- [ ] When there was an error in ajax call